### PR TITLE
Components Scaffolding

### PR DESCRIPTION
**Pull request by chethanagopinath**
Looks like this right now, based on the data

Screen Shot 2022-06-27 at 4 05 03 PM

Need to clean up a lot of the defaults

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```
Comment (jraigosaOC) [inline public/index.html:17]: maybe just 🔪 ?

Comment (jraigosaOC) [reply public/index.html:17]: since we're removing the file

File: src/Prompt.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    return (
+        <div>
+            {question}
+            {image}
+            {JSON.stringify(options)}
+        </div>
+    );
+}
+
+export default Prompt
```
Comment (jraigosaOC) [inline src/Prompt.js:13]: think the file structure is good for now.  I'm thinking we should a components directory under src and this should go in there. just to keep it a little tidier

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,60 @@
...
+        option: "Choose and move to 2",
+        nextId: 2
+      },
+      {
+        option: "Choose and move to 3",
+        nextId: 3
+      }
+    ]
+    }
+  ]
```
Comment (jraigosaOC) [inline src/App.js:51]: good w/ this for now.  think we might want to move this into a separate json data file.  there actually might a utility for turning flow charts into structured json.  will take a look.

**Review by jraigosaOC — COMMENTED** [review]
A few comments.  I think the only thing gating for me is adding a components folder w/ prompt.js in it.

File: public/index.html (hunk 1 above)
Comment (chethanagopinath) [reply public/index.html:17]: 👍🏼

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,14 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    questions.map((questionPrompt) => {
```
Comment (jraigosaOC) [inline src/App.js:9]: 👍

**Review by jraigosaOC — APPROVED** [review]
🎸

**Comment by chethanagopinath** [comment]
made some changes + addressed comments 

question - do we need the `robots.txt` as we aren't in scope of web scraping or anything for now 🙈 also do we need `reportWebVitals.js` and `setupTests.js`?  

feels like `package.json` also looks intense, but maybe we can clean that up later

may be good for a second glance (oooo :thank_you-8094: :juan-sunshine: !)

**Review by kelseyhuse30 — APPROVED** [review]

**Comment by chethanagopinath** [comment]
test

**Comment by meganweijiang** [comment]
> made some changes + addressed comments
> 
> question - do we need the robots.txt as we aren't in scope of web scraping or anything for now 🙈 also do we need reportWebVitals.js and setupTests.js?
> 
> feels like package.json also looks intense, but maybe we can clean that up later
> 
> may be good for a second glance (oooo :thank_you-8094: folks!)

I think we can worry about clean up later; the most important thing is getting the app working 😆

**Review by meganweijiang — APPROVED** [review]
❤️

**Pull request merged**